perception_and_utils/setup.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import setuptools
+
+setuptools.setup(
+    name="perception_and_utils",
+    version="0.0",
+    author="Priyam Parashar",
+    author_email="priyam8parashar@gmail.com",
+    description="SIRo friendly perception and utils for Aria and Spot",
+    packages=setuptools.find_packages(),
+    install_requires=[
+        "click",
+        "transformers",
+        # TODO: Add new dependencies here
```

Comment:
@zephirefaith what new installs would we need here?
Are you okay with the package name?

Reply:
Removed